src/kernel/src/interrupt.rs

```diff
@@ -147,7 +147,7 @@ pub fn handle_interrupt(number: u32) {
     let gi = get_global_interrupts();
     gi.ints[number as usize].raise();
     if number != 43 {
-        //logln!("external device interrupt {}", number);
```

Comment:
Is there any reason to keep this line at all?  If it's commented out, just remove it and let the VCS (git) remember.

Reply:
I've removed the line. It appears that line is commented out on main as well, but I see no issue with deleting it here.